metadata.json

```diff
@@ -20,7 +20,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 4.7.0 < 6.0.0"
+      "version_requirement": ">= 4.9.4 < 6.0.0"
```

Comment:
We had a discussion yesterday on the IRC about lowest version of puppet in modules with data-in-modules. We decided to go with 4.10.9 as there were soo many bug fixes for the hiera implementation.